docs/inheritance.md

```diff
@@ -98,3 +98,35 @@ updating only that node (e.g. with `oc edit`). In that case we'd see
 an `ansiblePort` set directly in that node's CR. This allows the user
 to change the `nodeTemplate` after creation and once reconciliation is
 completed all existing nodes will inherit the new value.
+
```

Comment:
Slight tangent. We should perhaps add a note that any top level property in the node overrides the whole property in the role. In other words - there is no need merging of any sub-properties.

This is by design the same as the way Ansible merges variables to help keep it intuitive and simple.